tools/azure-rest-api-specs-examples-automation/dotnet/test_main.py

```diff
@@ -1,6 +1,6 @@
 import unittest
-from main import break_down_aggregated_dotnet_example, format_dotnet
-
+import parameterized
```

Comment:
I added a `parameterized` package, and on my local, I have to `pip install parameterized`, do I need to do something here so that the pipeline could also have it?

Reply:
You can run the pip install in a step in CI

https://github.com/Azure/azure-sdk-tools/blob/a3d8c574bb81c6a99b6690d9562f95c47badc6d2/tools/azure-rest-api-specs-examples-automation/ci.yml#L38-L40